torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1327,3 +1343,608 @@ def _same_dtype_views(view1, view2):
         for x in post_processed_calling_convention_meta:
             assert x != -1
         return args_to_functionalization, post_processed_calling_convention_meta
+
+
+@dataclass
+class AutogradLazyBackwardCompileInfo:
```

Comment:
We won't have to cache this thing: instead on the cache path we can safely assert that the backward is already compiled, since we'll only save to the cache once the backward is compiled by inductor